artcommon/artcommonlib/konflux/konflux_db.py

```diff
@@ -58,11 +58,21 @@ def add_build(self, build: konflux_build_record.KonfluxBuildRecord):
         Insert a build record into Konflux DB
         """
 
+        def value_or_null(value):
+            if value is None:
+                return 'NULL'
+
+            elif isinstance(value, str):
+                return f"'{value}'"
```

Comment:
we can just do `return value` here?

Reply:
The query string will need to enclose every value we want to store between `''`, if that value is a string. That's why this is needed.